tests/test_database.py

```
@@ -0,0 +1,158 @@
+        with patch.object(db_module, 'init_database') as mock_init:
+            mock_engine = MagicMock()
+            mock_init.return_value = mock_engine
+            db_module._engine = None
+
+            def set_engine():
+                db_module._engine = mock_engine
+                return mock_engine
+
+            mock_init.side_effect = set_engine
+
+            engine = get_engine()
```

Ruff (0.14.10)

Local variable `engine` is assigned to but never used

Remove assignment to unused variable `engine`

(F841)